### PR TITLE
Japanese translations for DXP documentation

### PR DESCRIPTION
**Pull request by holatuwol**
The documentation that we translated is about a year old now (and a lot of it has probably been rearranged multiple times since then).

With that in mind, would it be possible to check for any quality issues?

### CODE REVIEW

**Comment by jrhoun** [comment]
Initial observations:

* Have to remove `docs\dxp\7.x\ja\README.md`, because its given rendering priority over `docs\dxp\7.x\ja\README.rst` which has the card rendering content.
* Once ^ is fixed, the cards are empty:
image
    * ^ because in `docs\dxp\7.x\ja\landing.html` all of the `name` params are empty
    image
* Saw two instances where markdown formatting was not rendered correctly because there wasn't a space between characters (there's probably more of these):
    image
    image
* Commented out content was still being shown, in some cases:
    image

**Comment by holatuwol** [comment]
In the latest force push, I've addressed the empty landing pages, the lack of italics, and I've made an attempt to address the problem with comments.

File: docs/dxp/7.x/ja/landing.html [hunk 1]
```diff
@@ -0,0 +1,258 @@
+<script type="module">
+	import Vue from 'https://cdn.jsdelivr.net/npm/vue@2.6.11/dist/vue.esm.browser.min.js'
+
+	const page = new Vue({
+		el: '#landingPage',
+		data: {
+			cards: [
+				{
+					icon: '#getting-started-logo',
+					sectionName: '==> getting_started.rst <==はじめに==> system-administration/installing-and-managing-apps/getting_started.rst <==はじめに==> using-search/getting_started.rst <==はじめに',
```
Comment (jrhoun) [inline docs/dxp/7.x/ja/landing.html:10]: @holatuwol this translation got messed up here.

**Comment by jrhoun** [comment]
After skimming through other sections, once my latest comment is resolved, I didn't see any other obvious issues.

File: docs/dxp/7.x/ja/landing.html (hunk 1 above)
Comment (holatuwol) [reply docs/dxp/7.x/ja/landing.html]: @jrhoun  Got it! Force pushed an update.

**Comment by jrhoun** [comment]
Merged. Thanks.

**Pull request closed without merging**